packages/web-frontend/src/tests/selectors/orgUnitSelectors.test.js

```diff
@@ -1,244 +1,258 @@
-        expect(
-          selectCurrentOrgUnit({ ...state, routing: { pathname: 'NOT_A_VALID_PATH' } }),
-        ).toEqual({});
-      });
-      it('Empty org unit is always shallowly equal', () => {
-        const orgUnit1 = selectCurrentOrgUnit({
-          ...state,
-          routing: { pathname: 'NOT_A_VALID_PATH #1' },
-        });
-        const orgUnit2 = selectCurrentOrgUnit({
-          ...state,
-          routing: { pathname: 'NOT_A_VALID_PATH #2' },
-        });
-        expect(orgUnit1).toBe(orgUnit2);
-      });
-    });
-
-    describe('selectOrgUnitChildren', () => {
-      it.skip('can select children of world', () => {
-        expect(selectOrgUnitChildren(state, 'World')).toContain(state.orgUnits.orgUnitMap.TO.TO);
-      });
-      it.skip('can select children of a project', () => {
-        expect(selectOrgUnitChildren(state, 'explore')).toContain(state.orgUnits.orgUnitMap.TO.TO);
-      });
-      it('can select children of country', () => {
-        expect(selectOrgUnitChildren(state, 'TO')).toContain(
-          state.orgUnits.orgUnitMap.TO.TO_Haapai,
-        );
-      });
-      it('can select children of facility', () => {
-        const children = selectOrgUnitChildren(state, 'TO_Haapai');
-        expect(children).toContain(state.orgUnits.orgUnitMap.TO.TO_HfevaHC);
-        expect(children).toContain(state.orgUnits.orgUnitMap.TO.TO_FoaMCH);
-      });
-      it('can select children of undefined', () => {
-        expect(selectOrgUnitChildren(state, undefined)).toEqual(undefined);
-      });
-    });
-
-    describe('selectOrgUnitSiblings', () => {
-      it('can select siblings of world', () => {
-        expect(selectOrgUnitSiblings(state, 'World')).toEqual([]);
-      });
-      it.skip('can select siblings of country', () => {
-        expect(selectOrgUnitSiblings(state, 'TO')).toEqual([state.orgUnits.orgUnitMap.PG.PG]);
-      });
-      it('can select siblings of district, including facility', () => {
-        expect(selectOrgUnitSiblings(state, 'PG_district_1')).toEqual([
-          state.orgUnits.orgUnitMap.PG.PG_district_2,
-          state.orgUnits.orgUnitMap.PG.PG_facility_1,
-        ]);
-      });
-      it('can select multiple siblings of facility', () => {
-        expect(selectOrgUnitSiblings(state, 'PG_facility_2')).toEqual([
-          state.orgUnits.orgUnitMap.PG.PG_facility_3,
-          state.orgUnits.orgUnitMap.PG.PG_facility_5,
-        ]);
-      });
-      it('can select siblings of facility with no siblings', () => {
-        expect(selectOrgUnitSiblings(state, 'PG_facility_4')).toEqual([]);
-      });
-      it('can select siblings of undefined', () => {
-        expect(selectOrgUnitSiblings(state, undefined)).toEqual([]);
-      });
-    });
-  });
-});
+/**
+ * Tupaia Web
+ * Copyright (c) 2020 Beyond Essential Systems Pty Ltd.
+ * This source code is licensed under the AGPL-3.0 license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import {
```

Comment:
Again, ESLint formatted this file (I believe just added blank lines between test cases)